__tests__/CollapsedMetrics.test.js

```diff
@@ -0,0 +1,611 @@
+    
+        expect(result).toBe(1.83);
+    });
+});
+
+// ROM
+
+describe('getROMs', () => {
+    
+        test('[] when empty', () => {
+            let set = {
+                reps: [],
+            };
+    
+            let result = sut.getROMs(set);
+    
+            expect(result).toEqual([]);
+        });
+    
+        test('returns roms from set with reps', () => {
+            let set = {
+                reps: [
+                    {
+                        isValid: true,
+                        removed: false,
+                        data: [-3456, 37, 1.833368, 200]
+                    }, 
+                    {
+                        isValid: true,
+                        removed: false,
+                        data: [-3456, 38, 1.821198, 250]
+                    }, 
+                    {
+                        isValid: true,
+                        removed: false,
+                        data: [-3456, 39, 1.08, 300]
+                    },
+                ]
+            }
+    
+            let result = sut.getROMs(set);
+    
+            expect(result).toEqual([200, 250, 300]);
+        });
+    
+        test('return [] when rep is invalid', () => {
+            let set = {
+                reps: [
+                    {
+                        isValid: false,
+                        removed: false,
+                    }
+                ]
+            }
+    
+            let result = sut.getROMs(set);
+    
+            expect(result).toEqual([]);
+        });
+    
+        test('return [] when rep is removed', () => {
+            let set = {
+                reps: [
+                    {
+                        isValid: true,
+                        removed: true,
+                    }
+                ]
+            }
+    
+            let result = sut.getROMs(set);
+    
+            expect(result).toEqual([]);
+    });        
+});
+
+describe('getAvgofROMs', () => {
+    
+        test('null when no reps', () => {
+            let set = {
+                reps: [],
+            }
+    
+            let result = sut.getAvgofROMs(set);
+    
+            expect(result).toBe(null);
+        });
+    
+        test('return avg of roms', () => {
+            let set = {
+                reps: [
+                    {
+                        isValid: true,
+                        removed: false,
+                        data: [-3456, 37, 1.833368, 200]
+                    }, 
+                    {
+                        isValid: true,
+                        removed: false,
+                        data: [-3456, 38, 1.821198, 250]
+                    }, 
+                    {
+                        isValid: true,
+                        removed: false,
+                        data: [-3456, 39, 1.08, 300]
+                    },
+                ]
+            }
+    
+            let result = sut.getAvgofROMs(set);
+    
+            expect(result).toBe("250.00");
```

Comment:
- [ ]  make it so it the average isn't one of the values, like make it actually mathed out